### PR TITLE
Reorganize Documentation

### PR DESCRIPTION
**Pull request by mgiammar**
- [ ] Seperate getting started from Simulator documentation
- [ ] Incorperate troubleshooting sections from #88

### CODE REVIEW

**Comment by deepanshs** [comment]
@mgiammar Can you remove the .svg file and use .png instead. Notebooks .ipynb will generate a lot of merge conflicts. It might be better to delete them from the PR. Once we are ready for release, we can bring them back in.

The PR has gone way too long. Let's merge and work on the remaining in a different PR.

File: docs/conf.py [hunk 1]
```diff
@@ -162,30 +170,30 @@
         "mrsimulator": None
     },
     "backreferences_dir": "examples",
-    "doc_module": ("mrsimulator"),
+    "doc_module": ("mrsimulator", "mrsimulator.methods"),
```
Comment (mgiammar) [inline docs/conf.py:173]: Remove ``"mrsimulator.methods"``

File: docs/installation/troubleshooting/package_manager.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Creating a Python environment using Anaconda
+""""""""""""""""""""""""""""""""""""""""""""
+
+Since different Python packages have different dependencies, installing multiple packages on the
+same machine issues. For example, ``mrsimulator`` requires at least ``numpy v1.17`` but
+``some-other-library`` might require exactly ``numpy v1.15``. These two libraries would likely
+throw errors when run in the same environment.
+
+For this reason, we recommend using an environment manager, like ``venv`` or ``anaconda``.
+We will look at anaconda for its simple commands. installation instructions can be found on the
```
Comment (mgiammar) [inline docs/installation/troubleshooting/package_manager.rst:13]: Capitalize ``instillation``

File: docs/installation/troubleshooting/python_errors.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+Common Python Syntax Errors
+===========================
+
+Python syntax is slightly different than other langues which cause some confusion.
+The following are some common syntax errors encountered and how to solve them.
+
+However, the easiest way to find and prevent syntax errors is to use a dedicated code editor.
+We recommend using VS Code on your local machine
+or Google Colab which runs everything online. These programs check for syntax errors as you
+write code
```
Comment (mgiammar) [inline docs/installation/troubleshooting/python_errors.rst:11]: Add period to end of sentence

File: docs/installation/troubleshooting/python_errors.rst [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+``IndentationError: expected an indented block`` means some code after a class/method/loop
+deceleration is missing an indent.
+
+``IndentationError: unindent does not match any outer indentation level`` means the code didn't
+return to a previous indentation level.
+
+``IndentationError: unexpected indent`` means Python encountered unexpected whitespace.
+
+Code blocks in Python rely on indentation levels (1 level = 4 spaces) so whitespace can't
+be placed randomly. code blocks are preceded by a ``:`` and all code in one
```
Comment (mgiammar) [inline docs/installation/troubleshooting/python_errors.rst:29]: Capitalize code

File: docs/installation/troubleshooting/python_errors.rst [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+.. code-block:: python
+
+    # Add numbers 0 through 9
+    total = 0
+    for i in range(10):
+        # New code block (4 spaces)
+        total += i
+    # Exit loop code block (0 spaces)
+    print(total)  # 45
```
Comment (mgiammar) [inline docs/installation/troubleshooting/python_errors.rst:42]: Add paragraph between ``code-block`` and next heading

File: docs/installation/users.rst [hunk 6]
```diff
@@ -59,15 +63,26 @@ On Google Colab Notebook
...
+To install the ``mrsimulator`` package, type
 
 .. code-block:: shell
 
       !pip install mrsimulator
 
-in the first cell, and execute. All done! You may now start using the library, or
-proceed to section :ref:`Introduction to Spin Systems <spin_systems_intro>` to continue
-the tutorial.
+in a first cell, and execute. All done! You may now start using the library, or
```
Comment (mgiammar) [inline docs/installation/users.rst:84]: Change to ``in a new cell``

File: docs/user_guide/method/method.rst [hunk 7]
```diff
@@ -0,0 +1,79 @@
+.. _method_documentation:
```
Comment (mgiammar) [inline docs/user_guide/method/method.rst:1]: This file will be updated to reflect proper method documentation in another pull request

File: docs/user_guide/methods_library/bloch_decay.rst [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+        ],
+    )
+
+The method may be imported using the following alias classes:
+
+.. code-block:: python
+
+  from mrsimulator.methods import BlochSpectrum
+
+.. minigallery:: mrsimulator.utils.spectral_fitting.make_LMFIT_params
```
Comment (mgiammar) [inline docs/user_guide/methods_library/bloch_decay.rst:31]: minigallery should be BlochDecaySpectrum, not fitting functions

File: docs/user_guide/methods_library/bloch_decay_CT.rst [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+            dict(
+                count=1024,
+                spectral_width=25e3,  # in Hz
+                reference_offset=-4e3,  # in Hz
+            )
+        ],
+    )
+
+The method may be imported using the following alias classes:
+hi
```
Comment (mgiammar) [inline docs/user_guide/methods_library/bloch_decay_CT.rst:26]: delete line

File: docs/user_guide/signal_processing/signal_processing.rst [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+
+.. code-block:: python
+
+    # Gaussian apodization along the first dimension (default)
+    sp.apodization.Gaussian(FWHM="10 Hz")
+
+    # Constant offset along the second dimension
+    sp.baseline.ConstantOffset(offset=10, dim_index=1)
+
+    # Lorentzian (exponential) along the first and third dimensions
```
Comment (mgiammar) [inline docs/user_guide/signal_processing/signal_processing.rst:131]: Change to exponential apodization. Lorentzian is a convolution

File: docs/user_guide/signal_processing/signal_processing.rst [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+        operations=[
+            sp.IFFT(),
+            sp.apodization.Gaussian(FWHM="25 Hz", dv_index=0),
+            sp.apodization.Gaussian(FWHM="70 Hz", dv_index=1),
+            sp.IFFT(),
+        ]
+    )
+
+The above list of operations will apply a step apodization as well as 25 and 70 Hz of Gaussian
+line broadening to dependent variables at index 0 and 1, respectively
```
Comment (mgiammar) [inline docs/user_guide/signal_processing/signal_processing.rst:162]: Period at end of sentence

File: src/mrsimulator/signal_processing/tests/test_apodization.py [hunk 12]
```diff
@@ -229,10 +230,10 @@ def test_scale_class():
 
 
 def test_Step_class():
```
Comment (mgiammar) [inline src/mrsimulator/signal_processing/tests/test_apodization.py:232]: rename to ``test_TopHat_class()``

File: docs/introduction/getting_started.rst [hunk 13]
```diff
@@ -0,0 +1,202 @@
...
+    C_site = Site(
+        isotope="13C",
+        isotropic_chemical_shift=100.0,  # in ppm
+        shielding_symmetric=SymmetricTensor(
+            zeta=70.0,  # in ppm
+            eta=0.5,
+        ),
+    )
+
+We now have two variables, ``H_site`` and ``C_site``, which are **Site** objects. ``H_site``
```
Comment (mgiammar) [inline docs/introduction/getting_started.rst:39]: reference site user documentation

File: docs/introduction/getting_started.rst [hunk 14]
```diff
@@ -0,0 +1,202 @@
...
+.. code-block:: python
+
+    # Import the BlochDecaySpectrum class
+    from mrsimulator.methods import BlochDecaySpectrum
+
+    # Create a BlochDecaySpectrum object
+    method = BlochDecaySpectrum(
+        channels=["13C"],
+        magnetic_flux_density=9.4,  # in T
+        rotor_angle=0.9553166,  # in rad (magic angle)
```
Comment (mgiammar) [inline docs/introduction/getting_started.rst:89]: Change to 54.736 * 3.1415 / 180

File: docs/user_guide/io/mrsim_IO.rst [hunk 15]
```diff
@@ -0,0 +1,277 @@
+.. _IO_documentation:
```
Comment (mgiammar) [inline docs/user_guide/io/mrsim_IO.rst:1]: Need to create api references to bolded objects

File: docs/user_guide/simulator/simulator.rst [hunk 16]
```diff
@@ -0,0 +1,338 @@
+.. _simulator_documentation:
```
Comment (mgiammar) [inline docs/user_guide/simulator/simulator.rst:1]: Need to create api references

**Review by deepanshs — DISMISSED** [review]

**Pull request merged**